### PR TITLE
adding docs

### PR DESCRIPTION
**Pull request by puja108**
WIP

### CODE REVIEW

**Comment by puja108** [comment]
RFR @webwurst 

we will need to resolve namespace and label issue maybe (see Slack) ;)

**Comment by puja108** [comment]
working on the namespace issue

File: manifests/registry-service.yaml [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+  namespace: kube-system
   labels:
-    app: registry
+    k8s-app: kube-registry
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "KubeRegistry"
 spec:
   type: NodePort
   ports:
   - name: registry
```
Comment (webwurst) [inline manifests/registry-service.yaml:11]: needs to be `kube-registry`

Comment (puja108) [reply manifests/registry-service.yaml:11]: right, thanks

**Comment by puja108** [comment]
so, as I remember us talking @webwurst we want to revert the namespace, I would then maybe put it into its own namespace. What do you think? `registry` or sth else?

**Comment by puja108** [comment]
Removed the additional foo and put the thing in a `registry` namespace, still somehow can't get it to work, not sure why. Could you maybe test @webwurst ?

**Comment by puja108** [comment]
tested by @webwurst and me

**Pull request merged**